pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "case-mapping"
```

Comment:
This name should be consistent with the primary source tree name, which at the moment is `case_builder`.  Should that directory be `case_mapping` instead?

IIRC, the hyphen here is appropriate.

Reply:
Removed with the directory rename below